naga/src/back/spv/index.rs

```diff
@@ -50,15 +50,16 @@ impl<'w> BlockContext<'w> {
     ///
     /// - An optional [`AccessIndex`], for case 2, applied to...
     /// - An optional [`Access`] or [`AccessIndex`], for case 3, applied to...
-    /// - A [`GlobalVariable`].
+    /// - A [`naga::GlobalVariable`].
```

Comment:
question: Why specify `naga::…` here? 🤔 Almost all the other Naga symbols around here aren't qualified, and I don't understand why this line merits an exception.

Reply:
Unfortunately, we have two types named `GlobalVariable` in play: the Naga IR's (`naga::GlobalVariable`) and the SPIR-V backend's (`naga::back::spv::GlobalVariable`). So pretty much every reference to `GlobalVariable` in the docs needs to be considered for ambiguity.

In this case, since we're talking about `naga::Expression`, it's probably clear enough, and the qualification is more of a distraction than a clarification.